avro_schema/frontend.lua

```diff
@@ -1,5 +1,13 @@
 -- The frontend.
--- Loads schema and generates IR.
+--
+-- Module overview:
+-- * `copy_schema` - validate and create schema
+-- * `copy_data` - validate a data by a schema; copy it
```

Comment:
Proposed wording: return copy of the data augmented with default values.

Reply:
ok. corrected.